src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -479,10 +511,13 @@ module.exports = function<T, P, I, TI, C, CX>(config : HostConfig<T, P, I, TI, C
 
     ReactCurrentOwner.current = null;
```

Comment:
While you're at it, this also doesn't belong in the hot path for the same reason.